manifests/init.pp

```diff
@@ -430,6 +430,7 @@
                                             '/var/log/cron',
                                           ]
         $default_logrotate_options      = [
+                                            'missingok',
```

Comment:
The section is `'RedHat', default: {` so it applies to RedHat and everything that is not explicitly called out above (Suse and Debian).